pom.xml

```diff
@@ -93,6 +93,14 @@
 
     <dependencyManagement>
         <dependencies>
+            <dependency>
+                <groupId>com.fasterxml.jackson</groupId>
+                <artifactId>jackson-bom</artifactId>
+                <version>${jackson.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
```

Comment:
@ali-ince Why this?

Reply:
@ali-ince I mean having this makes sense to remove any reference to Jackson deps like the kotlin module:

https://github.com/neo4j-contrib/neo4j-streams/blob/e900aa7242340763c190a14c470d124ebd6e1b64/pom.xml#L192-L202